### PR TITLE
Update romain ro (exemple)

### PR DESCRIPTION
**Pull request by AlExX2457**
They forgot to translate both words completely (exemple)

### CODE REVIEW

File: res/values-ro/strings.xml [hunk 1]
```diff
@@ -128,7 +128,7 @@
     <string name="Friend_Request_Sent">"Cerere de prietenie trimisă"</string>
     <string name="ERROR">"EROARE"</string>
     <string name="CONTINUE">"CONTINUĂ"</string>
-    <string name="Global_" formatted="false">"Global%"</string>
+    <string name="Global_" formatted="false">"Global formatat fals"</string>
```
Comment (Jeii20) [inline res/values-ro/strings.xml:131]: Wrong change here. You shouldn't translate and provide the `formatted="false"`.

**Review by Jeii20 — CHANGES_REQUESTED** [review]

File: res/values-ro/strings.xml (hunk 1 above)
Comment (AlExX2457) [reply res/values-ro/strings.xml:131]: Alright

Comment (AlExX2457) [reply res/values-ro/strings.xml:131]: So is not right, why, that said there Without%

Comment (AlExX2457) [reply res/values-ro/strings.xml:131]: There up at winner is Câstigator not how they translate there, can i get my tag translator or i need to try again and to see better there if is wrong or no something

Comment (Jeii20) [reply res/values-ro/strings.xml:131]: You just need to correct the changes

Comment (AlExX2457) [reply res/values-ro/strings.xml:131]: Alright, i changed right now

**Comment by AlExX2457** [comment]
Can you check now @Jeii20 (Thank you in advance)

**Comment by Jeii20** [comment]
PR closed. Another PR with requested change identified at #480.

**Review by Jeii20 — COMMENTED** [review]
.

**Pull request closed without merging**